docs/2.7.0/docs/daml-finance/tutorials/getting-started/holding.rst

```diff
@@ -0,0 +1,194 @@
+
+Sample code
+***********
+
+In order to show how this works in practice, let us now explore the ``Transfer`` script (used in the
+:doc:`Transfer <transfer>` chapter) step-by-step.
+
+Create ``Holding`` and ``Account`` Factories
+============================================
+
+The first instruction instantiates an account factory. This is just a template that is used by a
+party (the Bank in this case) to create accounts as part of the ``CreateAccount`` workflow.
+
+.. literalinclude:: /_templates/quickstart-finance/daml/Scripts/Holding.daml
+  :language: daml
+  :start-after: -- CREATE_ACCOUNT_FACTORY_BEGIN
+  :end-before: -- CREATE_ACCOUNT_FACTORY_END
+
+Notice how the ``ContractId`` is immediately converted to an interface upon creation: this is
+because our workflows do not have any knowledge of concrete template implementations.
+
+Similarly, we define a holding factory, which is used within an account to ``Credit`` and ``Debit``
+holdings.
+
+.. literalinclude:: /_templates/quickstart-finance/daml/Scripts/Holding.daml
+  :language: daml
+  :start-after: -- CREATE_HOLDING_FACTORY_BEGIN
+  :end-before: -- CREATE_HOLDING_FACTORY_END
+
+This factory contract can be used to create
+:ref:`Fungible <type-daml-finance-holding-fungible-factory-35358>` holdings, which are defined in
+:ref:`Daml.Finance.Holding.Fungible <module-daml-finance-holding-fungible-7201>`
+and are both :ref:`fungible <fungibility>`, as well as :ref:`transferable <transferability>`.
+
+We are adding a so-called *public party* as an observer to the holding factory. This is done to
+ensure that every other party has visibility over this contract, as all parties can `readAs` the
+public party. This is required when you want to to transfer the holding, which will be shown in the
+next tutorial.
+
+Open Alice’s and Bob’s Accounts
+===============================
+
+Once the factory templates are setup, we leverage our ``CreateAccount`` workflow to create accounts
+at the Bank for Alice and Bob.
+
+The creation of an account needs to be authorized by both the ``custodian`` and the ``owner``, i.e.,
+by the Bank and Alice in our case. Authorization is collected using an initiate / accept pattern.
+
+.. literalinclude:: /_templates/quickstart-finance/daml/Scripts/Holding.daml
+  :language: daml
+  :start-after: -- SETUP_ALICE_ACCOUNT_BEGIN
+  :end-before: -- SETUP_ALICE_ACCOUNT_END
+
+Bob’s account is created in a similar fashion.
+
+Create the Cash Instrument
+==========================
+
+In order to credit Alice’s account with some cash, we first introduce a cash
+:ref:`Instrument <type-daml-finance-interface-instrument-token-instrument-instrument-4350>`
+in our model.
+
+.. literalinclude:: /_templates/quickstart-finance/daml/Scripts/Holding.daml
+  :language: daml
+  :start-after: -- ISSUE_CASH_INSTRUMENT_BEGIN
+  :end-before: -- ISSUE_CASH_INSTRUMENT_END
+
+An instrument is a representation of what it is that we are holding against the bank. It can be as
```

Comment:
"Bank" or "bank". Sometimes we write with capital letter and sometimes not, maybe change this to make it consistent.